src/Servers/Kestrel/test/BindTests/AddressRegistrationTests.cs

```diff
@@ -129,12 +129,13 @@ public async Task RegisterAddresses_IPv6_Success(string addressInput, string[] t
         }
 
         [ConditionalTheory]
-        [MemberData(nameof(AddressRegistrationDataIPv6Port5000Default))]
+        [MemberData(nameof(AddressRegistrationDataIPv6Port5000And5001Default))]
         [IPv6SupportedCondition]
         [QuarantinedTest("https://github.com/dotnet/aspnetcore-internal/issues/2711")]
-        public async Task RegisterAddresses_IPv6Port5000Default_Success(string addressInput, string[] testUrls)
+        public async Task RegisterAddresses_IPv6Port5000And5001Default_Success(string addressInput, string[] testUrls)
         {
-            if (!CanBindToEndpoint(IPAddress.Loopback, 5000) || !CanBindToEndpoint(IPAddress.IPv6Loopback, 5000))
+            if ((!CanBindToEndpoint(IPAddress.Loopback, 5000) || !CanBindToEndpoint(IPAddress.IPv6Loopback, 5000)) &&
+                (!CanBindToEndpoint(IPAddress.Loopback, 5001) || !CanBindToEndpoint(IPAddress.IPv6Loopback, 5001)))
```

Comment:
When an agent is in this state, should the test fail and output information about what's holding the port?

Reply:
No. It just passes. Every AddressRegistrationTest that test the ability to bind to a specific port (as opposed to "port 0") has done this for years now.